anchore_engine/util/rpm.py

```diff
@@ -49,43 +49,72 @@ def split_rpm_filename(rpm_filename):
     return name, version, release, epoch, arch
 
 
-def compare_versions(pkg_a, ver_a, pkg_b, ver_b):
+def split_version(version):
```

Comment:
can we tweak the name to 'split_fullversion'. in these parsing packages 'version' does have a specific meaning and epoch is not a part of the version in that definition. Basically, I'd like to make it very clear where we deviate from standard rpm version behavior for the special cases we see comparing actual real-world data.